artemis-protocols/artemis-mqtt-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/mqtt/MQTTUtil.java

```diff
@@ -380,7 +380,7 @@ private static String formatByte(byte bite) {
    }
 
    private static String formatCase(String string) {
-      return CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.LOWER_CAMEL, string);
+      return CaseUtils.toCamelCase(string, false, ' ');
```

Comment:
This seems off? If its going from _UPPER_UNDERSCORE_ its unlikely to have space delimiters..seems more like it will end up with _lower_underscore_ after this? Or does the original also not do what it seems like its trying?

Reply:
fixed